terms.html

```diff
@@ -168,9 +189,12 @@
   </dd>
   <dt><dfn data-lt="verifier|verifiers|verifier's|credential verifiers|credential verifier's">verifier</dfn></dt>
   <dd>
-A role an <a>entity</a> performs by receiving one or more
-<a>verifiable credentials</a>, optionally inside a
-<a>verifiable presentation</a> for processing. Other specifications might refer
+A role an <a>entity</a> performs by requesting for, and receiving one or more
+<a>verifiable presentations</a> (from a <a>holder</a>), that contain
+<a>verifiable credentials</a> (or content derived from them) that are authored by <a>entities</a> that it trusts,
+so that they can reliably be processed for the purpose(s) they have been requested.
+The <a>verifiable credentials</a> may be received within a
+<a>verifiable presentation</a>. Other specifications might refer
 to this concept as a <dfn data-lt="relying parties">relying party</dfn>.
```

Comment:
```suggestion
A role an <a>entity</a> performs by requesting and/or receiving one or more
<a>verifiable presentations</a> (from a <a>holder</a>), that contain
<a>verifiable credentials</a> (or content derived from them) that are irrefutably authored by their <a>issuers</a>,
so that they can reliably be processed for the <a>verifier's</a> desired purpose(s).
The <a>verifiable credentials</a> may be received within a
<a>verifiable presentation</a>. Other specifications might refer
to this concept as a <dfn data-lt="relying parties">relying party</dfn>.
```

Reply:
This is outside the scope of this PR